src/plans/create/steps/migration-type/MigrationTypeStep.tsx

```diff
@@ -52,35 +58,69 @@ const MigrationTypeStep: FC = () => {
-                        </ExternalLink>
-                      </StackItem>
-                    </Stack>
-                  </HelpIconWithLabel>
-                }
-                description={t(
-                  'A warm migration migrates an active virtual machine (VM) from one host to another with minimal downtime.  This is not live migration.',
-                )}
-                value={migrationTypeField.value}
-                isChecked={migrationTypeField.value === MigrationTypeValue.Warm}
-                onChange={() => {
-                  migrationTypeField.onChange(MigrationTypeValue.Warm);
-                }}
-              />
+                        <StackItem>
+                          <ExternalLink isInline href={WARM_MIGRATION_HELP_LINK}>
+                            {t('Learn more')}
+                          </ExternalLink>
+                        </StackItem>
+                      </Stack>
+                    </HelpIconWithLabel>
+                  }
+                  description={t(
+                    'A warm migration migrates an active virtual machine (VM) from one host to another with minimal downtime.  This is not live migration.',
+                  )}
+                  value={migrationTypeField.value}
+                  isChecked={migrationTypeField.value === MigrationTypeValue.Warm}
+                  onChange={() => {
+                    migrationTypeField.onChange(MigrationTypeValue.Warm);
+                  }}
+                />
+              )}
+
+              {hasLiveMigrationProviderType(sourceProvider) && (
+                <Radio
+                  id={MigrationTypeValue.Live}
+                  name={MigrationTypeValue.Live}
+                  label={
+                    <HelpIconWithLabel label={migrationTypeLabels[MigrationTypeValue.Live]}>
+                      <Stack hasGutter>
+                        <StackItem>
+                          {t(
+                            'A live migration migrates an active virtual machine (VM) from one host to another with no downtime.',
+                          )}
+                        </StackItem>
+
+                        <StackItem>
+                          <ExternalLink isInline href={LIVE_MIGRATION_HELP_LINK}>
+                            {t('Learn more')}
+                          </ExternalLink>
+                        </StackItem>
+                      </Stack>
+                    </HelpIconWithLabel>
+                  }
+                  description={t(
+                    'A live migration migrates an active virtual machine (VM) from one host to another with no downtime.',
+                  )}
+                  value={migrationTypeField.value}
+                  isChecked={migrationTypeField.value === MigrationTypeValue.Live}
+                  onChange={() => {
+                    migrationTypeField.onChange(MigrationTypeValue.Live);
+                  }}
+                />
+              )}
```

Comment:
why it split for 2 checkes? seems like it can be 1 check.
Can u extract to a new component to reduce the noise from this component?